src/scripts/main.js

```diff
@@ -1,3 +1,368 @@
+    createElementInput.name === 'age' ||
+    createElementInput.name === 'salary'
+  ) {
+    createElementInput.type = 'number';
+  }
+
+  if (
+    createElementInput.name === 'name' ||
+    createElementInput.name === 'position'
+  ) {
+    createElementInput.type = 'text';
+  }
+
+  if (createElementInput.name === 'office') {
+    const select = document.createElement('select');
+
+    select.setAttribute('name', cleanText.toLowerCase());
+
+    const cities = [
+      'Tokyo',
+      'Singapore',
+      'London',
+      'New York',
+      'San Francisco',
+    ];
+
+    cities.forEach((city) => {
+      const option = document.createElement('option');
+
+      option.value = city;
+      option.textContent = city;
+      select.append(option);
+    });
+    createElementLabel.append(select);
+  }
+
+  if (createElementInput.name !== 'office') {
+    createElementLabel.append(createElementInput);
+  }
+});
+
+
+
+// можна звертатись по індексу
+// console.log(arrForLabel[0].textContent);
+
+// create Button Save
+const button = document.createElement('button');
+
+button.textContent = 'Save to table';
+
+createElementForm.append(button);
+
+button.addEventListener('click', function (e) {
+  e.preventDefault(); // Запобігаємо стандартній поведінці кнопки
+
+  const data = new FormData(createElementForm);
+  const namePerson = data.get('name');
+  const age = data.get('age');
+  const position = data.get('position');
+  const salary = data.get('salary');
+
+  if (namePerson.trim().length < 4) {
+    showError('Name value has less than 4 letters');
+  } else if (position.trim().length === 0) {
+    showError('Position field cannot be empty');
+  } else if (+age < 18 || +age >= 90 || isNaN(age)) {
+    showError('Age value is less than 18 or more than 90');
+  } else if (+salary <= 0) {
+    showError('Add salary');
+  } else {
+    showSuccessuf('Employee successfully added');
+
+    const newPerson = [
+      namePerson,
+      position,
+      data.get('office'),
+      age,
+      `$${(+(salary / 1000)).toFixed(3).replace('.', ',')}`,
+    ];
+
+    const newRow = tableBody.insertRow(-1);
+
+    newPerson.forEach((item, index) => {
+      const cell = newRow.insertCell(index);
+
+      cell.innerText = item;
+    });
+    createElementForm.reset();
+  }
+
+  setTimeout(() => {
+    const notifications = document.querySelectorAll('.notification');
+
+    notifications.forEach((notification) => {
+      notification.parentNode.removeChild(notification);
+    });
+  }, 3000);
+});
+
+
+// Функція для відображення повідомлень про помилки
+function showError(message) {
+  const createElementDiv = document.createElement('div');
+
+  createElementDiv.setAttribute('data-qa', 'notification.error');
+  createElementDiv.classList.add('notification', 'error');
+  createElementDiv.textContent = message;
+  document.body.append(createElementDiv);
+
+  setTimeout(() => {
+    createElementDiv.remove();
+  }, 2000);
+}
+
+function showSuccessuf(message) {
```

Comment:
There seems to be a typo in the function name `showSuccessuf`. It should likely be `showSuccess` to maintain consistency and clarity.